Makefile

```diff
@@ -84,3 +84,6 @@ dist: clean ## builds source and wheel package
 
 install: clean ## install the package to the active Python's site-packages
 	python setup.py install
+
+update-notebook-urls:
+	python pre-commit/notebook_urls.py examples/*
```

Comment:
I have changed `examples/**.ipynb` to `examples/*` as `rglob` was not reading files in subfolders properly.
I have also updated the description.

Reply:
I've updated this to use find and xargs to recursively find .ipynb files in examples so that we do not have to manually add every subdirectory.